src/main/java/com/internet/mixshop/model/Order.java

```diff
@@ -0,0 +1,39 @@
+package com.internet.mixshop.model;
+
+import java.util.List;
+
+public class Order {
+    private Long id;
+    private List<Product> products;
+    private Long userid;
+
+    public Order(List<Product> products, Long userid) {
+        this.id = id;
+        this.products = products;
+        this.userid = userid;
+    }
```

Comment:
`this.id = id;`
this doesn't make sense

also, you will not receive `products` from outside
let's initialize it as a new ArrayList

Reply:
done

Reply:
done